widget/bat.lua

```diff
@@ -32,19 +32,19 @@ local function factory(args)
     local settings  = args.settings or function() end
 
     bat_notification_critical_preset = {
-        title   = "Battery exhausted",
-        text    = "Shutdown imminent",
-        timeout = 15,
-        fg      = "#000000",
-        bg      = "#FFFFFF"
+        preset = naughty.config.presets.critical,
```

Comment:
For double safety, please check if this won't modify `naughty.config.presets.critical`.

Reply:
It doesn't, but this line doesn't seem to actually affect the notification's appearance at least for me. If you want I'll change it back to the old style or I could manually add  naughty.config.preset.critical's settings

Reply:
> this line doesn't seem to actually affect the notification's appearance at least for me

Are you referring to the added line? If so, what's the purpose of it?

Reply:
The purpose was to apply the settings in naughty.config.presets.critical [https://awesomewm.org/doc/api/libraries/naughty.html#config.presets] to the notification, but that doesn't seem to actually work inside the widget